SMLHelper/MonoBehaviours/EntitySpawner.cs

```diff
@@ -40,7 +40,7 @@ IEnumerator SpawnAsync()
             var lwe = obj.GetComponent<LargeWorldEntity>();
 
             // non-global objects cannot be spawned in unloaded terrain so we need to wait
-            if (lwe == null || lwe.cellLevel is not (LargeWorldEntity.CellLevel.Batch or LargeWorldEntity.CellLevel.Global))
+            if (lwe == null || lwe?.cellLevel is not (LargeWorldEntity.CellLevel.Batch or LargeWorldEntity.CellLevel.Global))
```

Comment:
This seems unnecessary - if the LHS evaluates to true (lwe == null), then the RHS will not be evaluated, so there will be no `NullReferenceException`.
```suggestion
            if (lwe == null || lwe.cellLevel is not (LargeWorldEntity.CellLevel.Batch or LargeWorldEntity.CellLevel.Global))
```